lbz/events/broker.py

```diff
@@ -2,42 +2,35 @@
 from typing import Callable, Dict, List
 
 from lbz.events.event import Event
+from lbz.handler_base import HandlerBase
 from lbz.misc import get_logger
 
 logger = get_logger(__name__)
 
 
-class EventBroker:
+class EventBroker(HandlerBase):
     def __init__(
         self,
         mapper: Dict[str, List[Callable]],
-        raw_event: dict,
+        event: dict,
         *,
+        context: object = None,
         type_key: str = "detail-type",
         data_key: str = "detail",
     ) -> None:
+        self.context = context
         self.mapper = mapper
-        self.event = Event(raw_event[data_key], event_type=raw_event[type_key])
+        self.event = Event(event[data_key], event_type=event[type_key])
 
     def handle(self) -> None:
```

Comment:
Shouldn't you start using `.react()` in `EventBroker` tests? Thanks to that, we will confirm this code uses `HandlerBase` behind the scenes 😉

Reply:
react in execution / tests but here its correct.

Reply:
I didn't suggest changing anything around this particular method, so that's ok 🙂